templates/rtx_6000_ada.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# GPU: 1x NVIDIA RTX 6000 Ada, 48 GB VRAM
+
+#cfg32=("resnet50,644" "resnext50,512" "unet_resnet50,440" "swin,240" "convnext,256")
+cfg16=("resnet50,1280" "resnext50,1024" "unet_resnet50,880" "swin,360" "convnext,500")
```

Comment:
run experiments to see if we wanna keep batch sizes as power of two 500->512

Reply:
Tried it and it results with OOM. Picked sizes will fill the memory 95%+.